subgraph/schema.graphql

```diff
@@ -110,4 +110,8 @@ type Receipt @entity {
   amountTransferred: BigInt
   "Total gas paid"
   gasTotal: BigInt!
+  "Refer to token id"
+  tokenId: BigInt
+  "Key owner of the receipt"
+  owner: String
```

Comment:
That's a bad idea as a single receipt can apply to multiple keys for multiple owners.